### PR TITLE
Import supports() clause.

### PR DESCRIPTION
**Pull request by chriseppstein**
http://www.w3.org/TR/css-cascade-4/#at-import

### CODE REVIEW

**Comment by chriseppstein** [comment]
This addresses issue #1754.

File: lib/sass/scss/parser.rb [hunk 1]
```diff
@@ -548,6 +561,15 @@ def supports_directive(name, start_pos)
         node(node, start_pos)
       end
 
+      def supports_clause
+        return unless tok(/supports\(/i)
+        ss
+        supports = supports_condition
+        ss
+        tok!(/\)/)
```
Comment (nex3) [inline lib/sass/scss/parser.rb]: If you make this a plain string (`")"`), it'll provide a better error message.

**Comment by nex3** [comment]
This needs SCSS and `sass-convert` tests. After that, it looks good.

File: lib/sass/scss/parser.rb (hunk 1 above)
Comment (chriseppstein) [inline lib/sass/scss/parser.rb]: @nex3 Using a string caused errors.

```
TypeError: wrong argument type String (expected Regexp)
    /Users/ceppstei/Projects/sass/lib/sass/scss/parser.rb:1193:in `scan'
    /Users/ceppstei/Projects/sass/lib/sass/scss/parser.rb:1193:in `tok'
    /Users/ceppstei/Projects/sass/lib/sass/scss/parser.rb:1103:in `tok!'
    /Users/ceppstei/Projects/sass/lib/sass/scss/parser.rb:569:in `supports_clause'
```

So I kept the regex

**Pull request merged**